api/src/modules/propositions/propositions.handler.ts

```diff
@@ -0,0 +1,144 @@
+import type { FastifyReply as Reply, FastifyRequest } from "fastify";
+import type { proposition } from "@src/types/proposition";
+import type Filters from "@src/types/Filters";
+import filtersFactorySlots from "@src/utils/filtersFactorySlots";
+
+type Request = FastifyRequest<{
+  Querystring: {
+    filter: Filters.Slot;
+  };
+  Params: {
+    id: number;
+  };
+}>;
+
+export const handleGetAllSlots = async (request: Request, reply: Reply) => {
```

Comment:
For the app we only need to reply with the proposition slots that are still available, among the next 10 slots.
These available slots are grouped in the next_propositions view you can use.

Reply:
We previously agreed to use parameters for specifik queries. That means default behavior is to reply with every rows.

Reply:
Ok my bad, I didn't catch up with the qs engine use.